### PR TITLE
Add aws release 15.0.0 with Kubernetes 1.20

### PR DESCRIPTION
**Pull request by MarcelMue**

### CODE REVIEW

**Comment by calvix** [comment]
taking over

**Comment by calvix** [comment]
cncf conformance merged https://github.com/cncf/k8s-conformance/pull/1423

**Comment by MarcelMue** [comment]
> cncf conformance merged [cncf/k8s-conformance#1423](https://github.com/cncf/k8s-conformance/pull/1423)

Damn @calvix - nice!

**Comment by calvix** [comment]
china is now happy as well

**Comment by MarcelMue** [comment]
> /retest

@njuettner It seems the test fails on a timeout - it takes longer than 1 hour to complete but the task abandons after an hour:
https://tekton.rfjh2.k8s.gorilla.eu-central-1.aws.gigantic.io/#/namespaces/test-workloads/pipelineruns/6cd7c4c9-c2a5-11eb-a64f-0e837194f2e2?pipelineTask=test-aws

**Comment by njuettner** [comment]
> > /retest
> 
> @njuettner It seems the test fails on a timeout - it takes longer than 1 hour to complete but the task abandons after an hour:
> https://tekton.rfjh2.k8s.gorilla.eu-central-1.aws.gigantic.io/#/namespaces/test-workloads/pipelineruns/6cd7c4c9-c2a5-11eb-a64f-0e837194f2e2?pipelineTask=test-aws

yeah, I just triggered it a few times to figure out how much we probably need to increase the time to get a good average :/

**Comment by njuettner** [comment]
/retest
ok increased the floor of timeout once again

**Comment by njuettner** [comment]
/retest
```{"caller":"github.com/giantswarm/awscnfm/v15/pkg/plan/executor.go:105","level":"error","message":"failed executing action `verify apps installed`","time":"2021-06-04T16:58:15.409768+00:00"}```

**Comment by njuettner** [comment]
@rossf7 excellent 👍🏻, I will do that

**Comment by njuettner** [comment]
There was a tagging issue, which has been fixed now. Need to run tests again

**Comment by njuettner** [comment]
I don't understand why it's failing, the tests are all passing

PipelineRun "42ce5bf3-cf63-11eb-925d-c2fb0b2915b8" failed to finish within "1h0m0s"

image

**Comment by njuettner** [comment]
```{"caller":"github.com/giantswarm/awscnfm/v15/pkg/plan/executor.go:105","level":"error","message":"failed executing action `verify master hostnetworkpod`","time":"2021-06-17T17:32:08.633562+00:00"}```

/retest

**Comment by njuettner** [comment]
We need to switch to flatcar LTS image, this was a agreement we made due to the unforeseen Kernel bug within the last release.

**Comment by MarcelMue** [comment]
Is there something we can do here to help? I assume the switch to LTS is happening?

**Comment by njuettner** [comment]
image

**Comment by njuettner** [comment]
@MarcelMue any idea where this 1h comes from? I don't see this number coming from `awscnfm` 🤔 . We also increased the timeout to 2h for the pipeline.

**Comment by njuettner** [comment]
Ok we're wait until we merge the S3 endpoint in `aws-operator`

**Comment by calvix** [comment]
I wannt to include  changes introduced in latest k8scloudconfig(more reliable pod evicting to prevent nodes to be dead under heavy load) so bumping aws-operator and returning test

**Comment by calvix** [comment]
all green, fine to release from my side

File: aws/v15.0.0/release.diff
Comment (alex-dabija) [inline aws/v15.0.0/release.diff:19]: I recently updated the realease.diff to compare this release with v14.2.2 and I noticed that we will be downgrading cert-manager from v2.7.1 to v2.7.0. Is this intentional or should we bump cert-manager to v2.7.1?

Comment (alex-dabija) [reply aws/v15.0.0/release.diff]: I upgraded cert-manager to 2.7.1. It only contains helm changes.

File: aws/v15.0.0/README.md
Comment (njuettner) [inline aws/v15.0.0/README.md:3]: We should also mention storage class `gp3` is becoming the standard by upgrading, before it was `gp2`

**Review by calvix — APPROVED** [review]

**Comment by MarcelMue** [comment]
I can't approve my own PR but looks good!

File: aws/v15.0.0/README.md
Comment (alex-dabija) [reply aws/v15.0.0/README.md]: I updated the highlights section to mention that `gp3` is now the default storage class.

**Review by alex-dabija — APPROVED** [review]

**Review by njuettner — APPROVED** [review]

**Pull request merged**
